hello-typescript/src/3-sort-objects-descending.ts

```diff
@@ -0,0 +1,28 @@
+export type Params = {
+    createdAt: string;
+    modifiedAt?: string;
+} | unknown;
+
+
+export function sortObjectsDescending(items: Params[]): Params[] {
+    const sortedItems: Params[] = items;
```

Comment:
بهتره ازش کپی بگیرید که آرایه‌ی اصلی تغییر نکنه
از طرفی تایپ رو خودش Infer میکنه

```suggestion
    const sortedItems = [...items];
```